src/components/molecules/ScheduleEvent/ScheduleEvent.scss

```diff
@@ -83,9 +122,64 @@ $description--margin-top: 2px;
     justify-content: center;
     align-items: center;
     padding: $bookmark--padding;
+    margin-left: auto;
 
     &:hover {
       padding-top: $bookmark-hover--padding-top;
     }
+
+    &--hidden {
+      display: none;
+    }
+  }
+
+  &__expand {
+    color: $concrete--dark;
+    outline: none;
+    border: none;
+    background-color: transparent;
+
+    &:focus {
+      outline: none;
+    }
+
+    &--square {
+      position: absolute;
+      height: $spacing--xl;
+    }
+
+    &--arrows {
+      height: $spacing--lg;
+      margin-top: $spacing--xs;
+
+      &--out {
+        display: none;
+      }
+    }
+
+    &--hidden {
+      display: none;
+    }
+
+    &--marged {
+      margin: auto;
+    }
+
+    &--padded {
+      padding-left: $ScheduleEvent--padding--left;
+      padding-right: $ScheduleEvent--padding--right;
+    }
+
```

Comment:
I feel we can move 
```
margin: auto;
```
to the base class name (`ScheduleEvent__expand`) and remove `--merged`, `--padded` selectors.

Reply:
we still need those for different container states hovered & not hovered